cnd/cnd.remote/tools/todo.txt

```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I would use the small ALv2 header here (https://www.apache.org/legal/src-headers.html#is-a-short-form-of-the-source-header-available)